benchmark-service/benchmark/templates/lookup.html

```diff
@@ -0,0 +1,70 @@
+{% extends "layout.html" %}
+{% block title %} Benchmarks {% endblock %}
+
+{% block content %}
+<link rel="stylesheet" href="{{ base_path }}/static/css/autocomplete.css" />
+
+<h1>Benchmark System</h1>
+
+<form autocomplete="off" action="{{ base_path }}/lookup" method="get">
+    <div class="autocomplete">
+        <input type="text" name="file" class="autoComplete"></input>
+    </div>
+    <button type="submit" id="button">Lookup</button>
+</form>
+
+{% if file is not none %}
+    <div class="attributes">
+        <div>SHA:
+            {% if benchmarks['sha'] is not none %}
+            {{ benchmarks['sha'] }}
+            {% else %}
+            unknown
```

Comment:
Can you confirm this is a direct copy of the old index.html?

Reply:
it isn't. The old index.html is now called lookup.html

Reply:
Right. So is this a carbon copy of index.html from before?